handbook/strategy/objectives.md

```diff
@@ -0,0 +1,25 @@
+# Launch Pad Objectives
+
+::: warning
+This page is a work in progress.
+:::
+
+We want to create a set of clear, measurable objectives for each school year, in rough order of priority. This helps hold us accountable to our goals, ensure that we are improving the club, and learn from our experiences.
+
+Objectives and results for each year are documented in this page. Each school year is denoted by the year of the fall semester, typically in September, and ends around May of the following year. Objectives have the following format:
+
+- **Objective: some overarching theme we want to work towards**
```

Comment:
Big fan of writing those goals down for accountability!

For discussion:
- I don't see any goals about the completion of the actual projects themselves (i.e I remember we said we wanted to focus on completeness for the projects)
- A bit handwavy, so feel free to ignore this, but how do we ensure that we are checking in on those goals?

Reply:
> I don't see any goals about the completion of the actual projects themselves (i.e I remember we said we wanted to focus on completeness for the projects)

ah thanks, _completely_ forgot to add that! added in ef906341fd31cdc4e234895ef87d066cc05fdc7a

> how do we ensure that we are checking in on those goals?

definitely something we need to address, but I added a note in eee7c58e71d1198905be56b98071a5e2ea4c8aad about following up (and hopefully we do that 😓 )

Reply:
I've opened https://github.com/ubclaunchpad/docs/issues/150 to address things we should do every term!